lib/_app/globalStyles.css

```diff
@@ -17,3 +17,19 @@ p {
 a {
   text-decoration: none;
 }
+
+.celeb-card img {
+  @apply scale-100 transform transition-all duration-[400ms];
+}
+
```

Comment:
Did you see this comment from the previous review? https://github.com/hollowverse/hollowverse/pull/69/files#r857912106